lib/models/organization.js

```diff
@@ -36,6 +36,9 @@ export default class Organization {
         if (this.attributes.legalNumber !== undefined && this.attributes.legalNumber !== null)
             org_object.lega_number = this.attributes.legalNumber;
 
+        if (this.attributes.timezone !== undefined && this.attributes.timezone !== null)
```

Comment:
you can use `isPresent`, [here](https://github.com/getlago/lago-nodejs-client/blob/main/lib/helpers/common.js#L1)

Reply:
Good catch!